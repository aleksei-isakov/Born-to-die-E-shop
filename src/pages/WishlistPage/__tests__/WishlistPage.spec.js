import { shallowMount, createLocalVue } from '@vue/test-utils';
import WishLlstPage from '../WishlistPage.vue';

describe('WishlistPage.vue', () => {
  let wrapper;

  beforeEach(() => {
    const localVue = createLocalVue();

    wrapper = shallowMount(WishLlstPage, {
      localVue
    });
  });

  it('renders a valid snapshot', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
