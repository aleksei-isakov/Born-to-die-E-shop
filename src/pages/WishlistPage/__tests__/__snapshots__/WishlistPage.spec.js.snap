// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WishListPage.vue renders a valid snapshot 1`] = `
<div
  class="wishlist__wrapper"
/>
`;

exports[`WishlistPage.vue renders a valid snapshot 1`] = `
<div
  class="wishlist__wrapper"
/>
`;
